ExampleMod/ExamplePlayer.cs

```diff
@@ -634,6 +635,103 @@ public class ExamplePlayer : ModPlayer
+		// You'll find the associated armor for this by searching for "ExampleBreastplate" in ExampleMod
+
+		// We start off by declaring a PlayerLayer, and give it our mods name, the layers name, its parent layer, and then a method that contains our drawing code
+		// This PlayerLayer we create will be added to the list of visible PlayerLayers in the ModifyDrawLayers hook below
+		public static readonly PlayerLayer ExampleBreastplateGlowmask = new PlayerLayer("ExampleMod", "ExampleBreastplateGlowmask", PlayerLayer.Body, delegate (PlayerDrawInfo drawInfo) {
+			
+			// We don't want the glowmask to draw if the player is cloaked or dead
+			if (drawInfo.shadow != 0f || drawInfo.drawPlayer.dead) {
+				return;
+			}
+
+			Player drawPlayer = drawInfo.drawPlayer;
+			Mod mod = ModLoader.GetMod("ExampleMod");
+
+
+			// To figure out when we should draw our ExampleBreastplate glowmask, we make use of GetEquipSlot.
+			// It takes in the name of the item that comes with an equip texture, and then its EquipType.
+			// By comparing that to drawPlayer.body, which represents the currently visible body armor, we make sure it's safe to draw ours
+			// Here we abort if the currently equipped body armor isn't what we need for our glowmask
+			if (drawPlayer.body != mod.GetEquipSlot("ExampleBreastplate", EquipType.Body)) {
+				return;
+			}
+
+			// Because we have a breastplate glowmask, and breastplates have different textures depending on if the player is male or female, we have to use two textures here
+			string gender = "";
+			if (!drawPlayer.Male) {
+				gender = "Female";
+			}
+
+			// The texture we want to display on our player
+			Texture2D texture = mod.GetTexture("Items/Armor/ExampleBreastplate_" + gender + "Body_Glowmask");
+
+			// The following variables (until drawData) are all copied 1:1 from vanilla code that handles breastplate drawing.
+			// It is advised to always copy the vanilla code that handles a particular EquipType, if you wish to make a glowmask for it,
+			// so it seamlessly overlaps
+			float drawX = (int)drawInfo.position.X + drawPlayer.width / 2;
+			float drawY = (int)drawInfo.position.Y + drawPlayer.height - drawPlayer.bodyFrame.Height / 2 + 4f;
+
+			// The rotation origin of the texture
+			// Because we are drawing a breastplate and it has the same alignment as the players' body, we use bodyOrigin
+			Vector2 origin = drawInfo.bodyOrigin;
+
+			// The position this texture will be drawn at on the screen
+			// Because we are drawing a breastplate and it has the same alignment as the players' body, we use bodyPosition
+			Vector2 position = new Vector2(drawX, drawY) + drawPlayer.bodyPosition - Main.screenPosition;
+
+			// The alpha of the texture when the player is blinking (immune)
+			// Should stay like that
+			float alpha = (255 - drawPlayer.immuneAlpha) / 255f;
+
+			// The color of the glowmask, pick what you want
+			Color color = Color.White;
```

Comment:
Not 'pick what you want' , most/all glowmasks will use a fully opaque white color because that's what glowmasks are supposed to do: they should be drawn at full brightness regular color regardless of light. It may be beneficial to put this kind of information in the comments.